### PR TITLE
Add EOL date for .NET 9

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

**Comment by mairaw** [comment]
Should we add end of support when the release is not supported yet?

**Comment by richlander** [comment]
We can bias towards transparency or a strict interpretation for preview support.

I suspect that 100% of users will appreciate this and 0% of users will be confused.

Somewhat related, I found this set of recommendations to be a good read: https://endoflife.date/recommendations

**Comment by richlander** [comment]
Oops! My mistake.

File: releases.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The following table lists in-support .NET releases.
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 9](release-notes/9.0/README.md) | November 12, 2024 | [STS][policies] | [9.0.0-preview.5][9.0.0-preview.5] |  |
+| [.NET 9](release-notes/9.0/README.md) | November 12, 2024 | [STS][policies] | [9.0.0-preview.5][9.0.0-preview.5] | November 9, 2027 |
```
Comment (richlander) [inline releases.md:13]:
```suggestion
| [.NET 9](release-notes/9.0/README.md) | November 12, 2024 | [STS][policies] | [9.0.0-preview.5][9.0.0-preview.5] | May 11, 2026 |
```

**Comment by richlander** [comment]
Slightly more useful reply @mairaw ...

I think of the EOL date as being for the major version while our support policies cover what the support is for preview releases.

https://github.com/dotnet/core/blob/main/release-policies.md#support-phases

So, if a user concluded that Preview 5 was supported until the given date, then they would come to the wrong conclusion per our published policies. I suspect few users would have that challenge.

**Comment by mairaw** [comment]
I still find a bit misleading that .NET 9 shows up on this page under supported versions.

**Comment by richlander** [comment]
Ah. That part is a mistake. It should be under "Preview" versions. That sections keeps on coming (at P1) and going (at RC1). I can fix that.

**Comment by richlander** [comment]
https://github.com/dotnet/core/pull/9363

**Pull request merged**